### DISCUSSION TITLE
headlines in the documentation should be anchored

### DISCUSSION THREAD

**Discussion by c33s** [Codeception/Codeception#6133] [General]
just wanted to link to "customization > events" where i would assume that i can click on the headline to get a link like
https://codeception.com/docs/08-Customization#events (without allowing cookies or tons of javascript if that is currently required for this regular html feature) but clicking on the headlines does nothing.

would be good to build the documentation  so this is possible. what do you think?

**Comment by ThomasLandauer** [comment]
The headers already have `id`s :-) So what you mean is to add a `#` character next to the headline, to allow easy copying of the full path, right?
